Formula/clp.rb

```diff
@@ -0,0 +1,125 @@
+class Clp < Formula
+  desc "Linear programming solver"
+  homepage "https://projects.coin-or.org/Clp"
+  url "https://www.coin-or.org/download/pkgsource/Clp/Clp-1.17.2.tgz"
```

Comment:
It seems that this link is not available. Can we use the one in github,  https://github.com/coin-or/Clp/archive/releases/1.17.2.zip ?

Reply:
Their whole website is down, actually. I'll see if it comes back up this morning.

Reply:
FYI, here is a version which uses GitHub links https://github.com/dreal-deps/homebrew-ibex/blob/master/clp%401.17.rb